inspector/src/components/actionTabs/tabs/propertyGrids/animations/animationCurveEditorComponent.tsx

```diff
@@ -1,28 +1,406 @@
 import * as React from "react";
 import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
 import { faTimes } from "@fortawesome/free-solid-svg-icons";
+import { Animation } from 'babylonjs/Animations/animation';
+import { EasingFunction, Vector4, Vector2, IAnimationKey, IEasingFunction} from 'babylonjs';

```

Comment:
to allow treeshacking this must be imported from the complete path (like Vector2 from maths.vector,...)

Reply:
thank you :)